lib/pbench/server/api/resources/__init__.py

```diff
@@ -837,9 +869,9 @@ def validate(self, json_data: JSONOBJECT) -> JSONOBJECT:
             raise MissingParameters(bad_keys)
 
         processed = {}
-        for p in json_data:
-            tp = self.parameters.get(p)
-            processed[p] = tp.normalize(json_data[p]) if tp else json_data[p]
+        for k, v in json_data.items():
+            tp = self.parameters.get(k)
+            processed[k] = tp.normalize(v) if tp else v
```

Comment:
Nice.

Reply:
Yeah, I actually added some instrumentation here trying to figure out why my `mine` was getting lost, and noticed that a few seconds spent refactoring would make it look a lot nicer.